packages/@okta/vuepress-site/code/react/okta_react_sign-in_widget/index.md

```diff
@@ -264,18 +270,16 @@ export default withRouter(class AppWithRouterAccess extends Component {
     //
     // pkce={false}
```

Comment:
This comment block needs to move above the `config` declaration at line ~250

Reply:
Done!